rasa/telemetry.py

```diff
@@ -573,6 +573,18 @@ def strip_sensitive_data_from_sentry_event(
     for value in event.get("exception", {}).get("values", []):
         for frame in value.get("stacktrace", {}).get("frames", []):
             frame["abs_path"] = ""
+
+            if "site-packages" in frame["filename"]:
+                # drop site-packages and following slash / backslash
+                frame["filename"] = frame["filename"].split("site-packages")[-1][1:]
```

Comment:
could split using `site-packages/` which would make it more obvious, but that's a detail

Reply:
well, that won't work on windows due to different path separators... 😂

Reply:
well, needs to work for windows as well and `\`... 😉

Reply:
oh that's a good point!